src/pages/TCY/queries/useTcyClaims.tsx

```diff
@@ -59,6 +62,11 @@ export const useTCYClaims = (accountNumber: number | 'all') => {
             if (!isUtxoChainId(fromAccountId(accountId).chainId))
               return [fromAccountId(accountId).account]
 
+            const isMetaMaskMultichainWallet = wallet instanceof MetaMaskMultiChainHDWallet
+
+            // Metamask snap might be uninstalled but UTXO accounts not cleared yet because of reactivity
+            if (isMetaMaskMultichainWallet && !isSnapInstalled) return []
+
```

Comment:
Seems weird but as this hook is fired asap, we need to check if the snap is installed because accounts can yet exists in the store until the reactivity hits and clear them, this is the only place causing issue I've been able to find, and it's an edge cases as it means you uninstalled the snap while not being on shapeshift, but needed!